### DISCUSSION TITLE
Any interest of adding a video player to Blazorise?

### DISCUSSION THREAD

**Discussion by sturlath** [Megabit/Blazorise#3264] [Q&A] [answered]
If so google's [Shaka Player ](https://github.com/google/shaka-player) would be a good choice. 

There seam to  be no real **streaming** players out there for Blazor [it seams](https://docs.microsoft.com/en-us/answers/questions/377441/can-you-use-the-azure-media-player-in-a-blazor-was.html).

But **if not**, is there any chance you could you point me in the direction of adding one my self to Blazor? 

I just have no idea where to begin consuming something like this into Blazor.

**Comment by stsrki** [comment, marked as answer]
Seems to me it could be used as any almost any other JS library: https://shaka-player-demo.appspot.com/docs/api/tutorial-basic-usage.html

We could do it, although it is not a priority for now. The soonest it could be for v1.2. If you want to make it then you can look at how we created our own components that use third-party libs. Search DatePicker, TimePicker, Tooltip, etc. In all of them, you need to have JSInterop that calls javaScript function to `initialize` and `destroy` JS instance. In some cases, you would need to invoke C# methods. The process is fairly simple but it can always tend to be harder.

Going on the TODO list.

**Reply by sturlath** [reply]
Awesome to have it on the list! I'll give it a go and try my self. I'll try to follow your code and if I manage to have something working I'll of course share it with you.

**Reply by sturlath** [reply]
I created a [draft](https://github.com/Megabit/Blazorise/pull/3267). Hope its something you can use or I can help with finishing.
